aries-site/src/pages/foundation/icons.js

```diff
@@ -17,9 +31,93 @@ const Icons = () => {
         canonicalUrl="https://design-system.hpe.design/foundation/icons"
       />
       <ContentSection>
-        <Subsection name={title} level={1} topic={topic} />
```

Comment:
I'm thinking we can add to the first sentence under the Icons heading: 

Icons are an integral part of our visual storytelling style to represent objects, actions, programs and other communicative symbols. This clean, dynamic style remains flexible while retaining the voice of our brand.